### PR TITLE
Consume Remixes + Stems API Endpoints

### PR DESCRIPTION
**Pull request by piazzatron**
### Trello Card Link
https://trello.com/c/A7BOsv5J

### Description
Consume stems, remixes, remixing endpoints. 

### Dragons

None

### How Has This Been Tested?

Tested on remix pages, tracks with stems.

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/piazz-remix-stems-api-2

File: src/services/AudiusBackend.js [hunk 1]
```diff
@@ -685,58 +685,6 @@ class AudiusBackend {
     }
   }
 
-  static async getStems({ trackId }) {
```
Comment (raymondjacobson) [inline src/services/AudiusBackend.js:688]: satisfying

File: src/services/audius-api-client/AudiusAPIClient.ts [hunk 2]
```diff
@@ -31,14 +33,17 @@ const ENDPOINT_MAP = {
   userRepostsByHandle: (handle: OpaqueID) => `/users/handle/${handle}/reposts`,
   getPlaylist: (playlistId: OpaqueID) => `/playlists/${playlistId}`,
   topGenreUsers: '/users/genre/top',
-  track: (trackId: OpaqueID) => `/tracks/${trackId}`
+  track: (trackId: OpaqueID) => `/tracks/${trackId}`,
```
Comment (raymondjacobson) [inline src/services/audius-api-client/AudiusAPIClient.ts:36]: should we call the getTrack, getStems, etc? to be consistent? or maybe rename the others

**Review by raymondjacobson — APPROVED** [review]

File: src/services/audius-api-client/AudiusAPIClient.ts (hunk 2 above)
Comment (piazzatron) [reply src/services/audius-api-client/AudiusAPIClient.ts:36]: will add verb

File: src/services/AudiusBackend.js (hunk 1 above)
Comment (piazzatron) [reply src/services/AudiusBackend.js:688]: you love to see it

**Pull request merged**